### ISSUE TITLE
Add iframe and browser Window with in the godot application

### ISSUE THREAD

**Issue by Bibekz-dev** [godotengine/godot-proposals#1752]
Sience Godot is available on web too and also has a http request i think a iframe and Opening browser window within godot game will be a goood idea like creating html editor or python editor by using javascript brython or any other ide. it will also make godot better option for not only games but apps and softwares too. and dear sir if you have any problems with it pllease tell me :( i will be sad and its ok Tears of joy :"')

**Comment by Calinou** [comment]
In the interest of making high-quality proposals, please [read the rules](https://github.com/godotengine/godot-proposals#readme) and fill in the issue template. This proposal will be reopened once this is done :slightly_smiling_face:

**Comment by Bibekz-dev** [comment]
> Duplicate of #960.
> 
> PS: When opening proposals, please follow the [template](https://github.com/godotengine/godot-proposals/blob/master/.github/ISSUE_TEMPLATE/feature---enhancement-proposal.md) instead of replacing it.

sir done please

**Comment by Bibekz-dev** [comment]
and sorry i didnt found it and thought it wasnt there
